app/app.go

```diff
@@ -1866,6 +1867,10 @@ func (app *App) Start(w io.Writer, process string) error {
 	err = prov.Start(app, process)
 	if err != nil {
 		log.Errorf("[start] error on start the app %s - %s", app.Name, err)
+		if _, ok := err.(*docker.ContainerAlreadyRunning); !ok {
```

Comment:
We shouldn't have docker specific errors in the `app` package as especially since this is only implemented in the `provision/docker` provisioner and would not work as expected when using kubernetes or swarm provisioners.
If we're to handle already running errors in the `app` package a new error could be created in the `provision` package, something like `provision.ErrAppAlreadyStarted` and all provisioners would be able to return it.